e2e/cypress/support/api_commands.js

```diff
@@ -494,6 +494,24 @@ Cypress.Commands.add('apiSaveThemePreference', (value = JSON.stringify(theme.def
     });
 });
 
+/**
+ * Saves the preference over link preview
+ * This API assume that the user is logged in and has cookie to access
+ * @param {boolean} value - theme object.  Will pass default value if none is provided.
+ */
+Cypress.Commands.add('apiSaveLinkPreviewPreference', (showExpanded = false) => {
+    return cy.getCookie('MMUSERID').then((cookie) => {
+        const preference = {
+            user_id: cookie.value,
+            category: 'display_settings',
+            name: 'link_previews',
+            value: showExpanded,
+        };
+
+        return cy.apiSaveUserPreference([preference]);
+    });
+});
+
```

Comment:
Please add another API command for `cy.apiSaveImagePreviewPreference()`
```javascript
// @param {boolean} collapsed - Either "true" to collapse image preview (default), or "false"

Cypress.Commands.add('apiSaveImagePreviewPreference', (collapsed = 'false') => {
  ...
});
```

Reply:
I have checked and I made a mistake. The two different options we have in terms of settings is whether we collapse or expand all the previews (no matter if image or link) (collapse_previews) and whether we show the preview or not at all (no matter if image or link) (link_preview). I will create the function for both, and call both on startup of the test.

Reply:
It's not true; they have separate setting. `link_previews` is for link and `collapse_previews` is for image.